packages/core/saved-objects/core-saved-objects-server-internal/src/routes/create.ts

```diff
@@ -48,6 +50,7 @@ export const registerCreateRoute = (
       },
     },
     catchAndReturnBoomErrors(async (context, req, res) => {
+      logger.warn("The create saved object API '/api/saved_objects/{type}/{id}' is deprecated.");
```

Comment:
```suggestion
      logger.warn("Detected use of saved object HTTP API 'POST /api/saved_objects/{type}/{id}'. This endpoint is deprecated and will be removed soon.");
```

Reply:
In my suggestion I restructured the warning message slightly trying to optimise for:

* Counting: If we prefix every log with the same message "Detected use of saved object HTTP API" it will be easier to query Kibana logs and get a count.
* Added the fully qualified endpoint description by adding `POST` in front of the path definition.
* Tried to make the message sound a bit more scary 😄 

Let me know what you think @TinaHeiligers !

Reply:
I see what you're saying and your suggestion does make deprecation logs for these API's more consistent with the two we already have. Since we don't have a 'replacement' we can't use the exact same log message as the `legacy_import_export` routes, what about:
```
logger.warn(
    "The create API '/api/kibana/saved_objects/{type}/{id}' is deprecated. This endpoint will be removed soon"
)
```
?
I'll change the log messages to ☝️ .

Reply:
>for similar [log messages](https://github.com/elastic/kibana/blob/2749e95fe0a5f0d1617129765eb3b84ea0cec097/packages/core/saved-objects/core-saved-objects-server-internal/src/routes/legacy_import_export/import.ts#L47-L49)

Thanks for linking to this, I was not aware that we'd done something similar.

> We don't know for certain that we can remove these API's, so adding a second qualifier is risky if we change our minds

It's a little scary to think they may be around in some form, but I take your point that we don't need to commit one way or the other in the message.

In the pre-existing message you link to a suggestion is made about an alternative. I think a second sentence giving some guidance would be really nice, but I'm struggling to think of something. Maybe:

"Do not rely on saved object APIs for your functionality, instead use plugin-specific endpoints."